README.md

```diff
@@ -27,6 +27,12 @@ The collection supports RHEL in-place upgrades for the following RHEL versions:
 
 The collection may be used for the RHEL upgrade paths and minor versions supported by the indicated upgrade utilities (Leapp or RUT). Refer the to Red Hat knowledge solution article [Supported in-place upgrade paths for Red Hat Enterprise Linux](https://access.redhat.com/articles/4263361) for the latest support details.
 
+Third-party products and packages are not upgraded. It is expected that the playbook including the `upgrade` role will perform the additional tasks required to handle the upgrade of any installed third-party tools and agents, for example [Veritas Cluster](https://www.veritas.com/support/en_US/doc/infoscale_wp_upgradewithRedHat), [SAP HANA](https://access.redhat.com/solutions/5154031), etc. Likewise for packages installed from non-RHEL repositories such as [Red Hat Software Collections](https://access.redhat.com/support/policy/updates/rhscl), [EPEL](https://docs.fedoraproject.org/en-US/epel/), [RPM Fusion](https://rpmfusion.org/), etc.
+
+Having said that, most application workloads will still function correctly after a RHEL in-place upgrade if simply left untouched, thanks to [RHEL Application Compatibility](https://access.redhat.com/articles/rhel8-abi-compatibility) support.
```

Comment:
most application workloads sounds mysterious and questionable.

Reply:
Fair enough. Is this better?... 

Having said that, many application workloads will benefit from [RHEL Application Compatibility](https://access.redhat.com/articles/rhel8-abi-compatibility) support such that they will still function correctly after a RHEL in-place upgrade if simply left untouched. Of course, the only way to know for sure is to run a test upgrade and then assess if there is any unexpected impact to your app. Pro tip: Test in your lower environments before moving on to production.

Reply:
Yes, it's better. However, if the 3rd party SW is supported on the target version, does it really matter how that target version has been 'created', by the a fresh install or by the upgrade? I like a simple and clear line in the leapp doc: "The installed applications and utilities, along with the configurations and preferences, are incorporated into the new version. " from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/upgrading_from_rhel_7_to_rhel_8/index#ref_key-migration-terminology_upgrading-from-rhel-7-to-rhel-8

Reply:
Removed mysterious wording and moved under "Not in scope" section.